src/viam/app/client.py

```diff
@@ -0,0 +1,48 @@
+from typing_extensions import Self
+
+from grpclib.client import Channel
+
+from viam.rpc.dial import DialOptions, _dial_app, _get_access_token
+from viam import logging
+from viam.app.data.client import DataClient
+
+LOGGER = logging.getLogger(__name__)
+
+
+class AppClient:
+    """gRPC client for all communication and interaction with app.
+
+    Use create() to instantiate an AppClient::
+
+        AppClient.create(...)
+
+    You may ``close()`` an AppClient after its use.
```

Comment:
I think we should probably get rid of this line. It should be apparent by the existence of a `close()` method that it can be called, and any considerations for when it should or shouldn't be used probably better belongs in the `close()` documentation.